packages/core/test-e2e/insured-bridge/e2e_InsuredBridge.js

```diff
@@ -0,0 +1,118 @@
+    await L1_ERC20.deployTransaction.wait();
+
+    await L1_ERC20.addMember(1, l1Wallet.address);
+    await L1_ERC20.mint(l1Wallet.address, 1234);
+
+    // Deploy the paired ERC20 token to L2. This takes in the address of the L2 bridge and the associated L1 token.
+    L2_ERC20 = await factory__L2_ERC20
+      .connect(l2Wallet)
+      .deploy(predeploys.OVM_L2StandardBridge, L1_ERC20.address, "L2 ERC20", "L2T");
+
+    await L2_ERC20.deployTransaction.wait();
+
+    // Connect to the L2 standard bridge from the predeploys.
+    L2StandardBridge = factory__L2StandardBridge.connect(l2Wallet).attach(predeploys.OVM_L2StandardBridge);
+
+    // Fetch the pre-deployed L1 standard bridge address from the predeploy.
+    const L1StandardBridgeAddress = await L2StandardBridge.l1TokenBridge();
+    L1StandardBridge = factory__L1StandardBridge.connect(l1Wallet).attach(L1StandardBridgeAddress);
+  });
+
+  describe("Basic bridging functionality", async () => {
+    it("Can send tokens from L1 to L2", async () => {
+      // Wallet should have to tokens on L1 from the mint action and no tokens on L2.
+      assert.equal((await L1_ERC20.balanceOf(l1Wallet.address)).toString(), "1234");
+      assert.equal((await L2_ERC20.balanceOf(l2Wallet.address)).toString(), "0");
+
+      // Allow the gateway to lock up some of our tokens.
+      await L1_ERC20.approve(L1StandardBridge.address, 1234);
+      // await tx1.wait();
+
+      // Lock the tokens up inside the gateway and ask the L2 contract to mint new ones.
+      const depositTx = await L1StandardBridge.depositERC20(L1_ERC20.address, L2_ERC20.address, 1234, 2000000, "0x");
+      await depositTx.wait();
+
+      // Wait for the message to be relayed to L2.
+      const [msgHash1] = await watcher.getMessageHashesFromL1Tx(depositTx.hash);
+
+      // Fetch the L2 transaction receipt. This also acts to block until the bridging transaction has been mined.
+      const receipt = await watcher.getL2TransactionReceipt(msgHash1, true);
+      // Deposit transaction should be sent to the OVM_L2CrossDomainMessenger on L2 and originate from 0x00 on L2.
+      assert.equal(receipt.to, predeploys.OVM_L2CrossDomainMessenger);
+      assert.equal(receipt.from, ZERO_ADDRESS);
+
+      // The balances should have incremented on L2.
+      assert.equal((await L1_ERC20.balanceOf(l1Wallet.address)).toString(), "0");
+      assert.equal((await L2_ERC20.balanceOf(l2Wallet.address)).toString(), "1234");
+    });
+    it("Can send tokens back from L2 to L1", async () => {
+      // Burn the tokens on L2 and ask the L1 contract to unlock on our behalf.
+      const withdrawTx = await L2StandardBridge.withdraw(L2_ERC20.address, 1234, 0, "0x");
+      await withdrawTx.wait();
+
+      // Wait for the message to be relayed to L1. Check addresses in the logs match the L1 token and bridge.
+      const [msgHash2] = await watcher.getMessageHashesFromL2Tx(withdrawTx.hash);
```

Comment:
Is this transfer instant because its on a test node?

Reply:
yup